modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/DataLayoutResourceTest.java

```diff
@@ -349,6 +366,40 @@ public void testPostDataLayoutContext() throws Exception {
 		Assert.assertEquals(200, httpResponse.getStatusCode());
 	}
 
+	@Override
+	@Test
+	public void testPutDataLayout() throws Exception {
+		super.testPutDataLayout();
+
+		DataLayout postDataLayout = testPutDataLayout_addDataLayout();
+
+		DataLayout randomDataLayoutWithVisualProps =
+			_createRandomDataLayoutFieldsWithVisualProps();
+
+		DataLayout putDataLayoutWithVisualProps =
+			dataLayoutResource.putDataLayout(
+				postDataLayout.getId(), randomDataLayoutWithVisualProps);
+
+		assertEquals(
+			randomDataLayoutWithVisualProps, putDataLayoutWithVisualProps);
+		assertValid(putDataLayoutWithVisualProps);
+		Assert.assertTrue(
+			equals(
+				randomDataLayoutWithVisualProps.getDataLayoutFields(),
+				putDataLayoutWithVisualProps.getDataLayoutFields()));
+
+		DataLayout getDataLayoutWithVisualProps =
```

Comment:
I think this can be just `getDataLayout` rigth?

Reply:
You're right! Thanks.